controller-server/src/main/java/com/yahoo/vespa/hosted/controller/application/JobStatus.java

```diff
@@ -157,27 +138,45 @@ public boolean equals(Object o) {
     public static class JobRun {
 
         private final long id;
-        private final Version version;
-        private final ApplicationVersion applicationVersion;
+        private final Version platform;
+        private final ApplicationVersion application;
+        private final Optional<Version> sourcePlatform;
+        private final Optional<ApplicationVersion> sourceApplication;
         private final String reason;
         private final Instant at;
 
-        public JobRun(long id, Version version, ApplicationVersion applicationVersion, String reason, Instant at) {
+        public JobRun(long id, Version platform, ApplicationVersion applicationVersion, Optional<Version> sourcePlatform, Optional<ApplicationVersion> application, String reason, Instant at) {
```

Comment:
Should rename the `ApplicationVersion` parameter as well.

Reply:
Of course!